conda/base/constants.py

```diff
@@ -42,6 +42,12 @@
     )
 
 SEARCH_PATH += (
+    '$XDG_CONFIG_HOME/conda/.condarc',
+    '$XDG_CONFIG_HOME/conda/condarc',
+    '$XDG_CONFIG_HOME/conda/condarc.d/',
+    '~/.config/conda/.condarc',
+    '~/.config/conda/condarc',
+    '~/.config/conda/condarc.d/',
```

Comment:
For consistency, I would suggest moving the XDG directories after `$CONDA_ROOT` and before `~/.conda`; that way, the install directory is always searched before $HOME and its subdirectories.

Reply:
Done with that, I've moved the XDG directories after `$CONDA_ROOT` and before `~/.conda` for consistency.

Reply:
Thanks for making the change. PR looks good to go now.